pythonx/jedi_vim.py

```diff
@@ -370,9 +370,11 @@ def annotate_description(d):
     return '[%s] %s' % (typ, code)
 
 
-def show_goto_multi_results(definitions):
+def show_goto_multi_results(definitions, mode):
```

Comment:
Can you maybe add a quick comment here why the different modes are needed? I'm not 100% sure why there is a different behavior for usages/goto. It might make a lot of sense, but when reading the code I wasn't sure why you implemented it that way.

Reply:
It gets passed through to `jedi#add_goto_window`, where it triggers different behavior.

Reply:
Can we use for_usages here as well? That would be way clearer.

Reply:
There are different modes here after all (had that first, and then reverted), and the string gets used in the qf title.

Reply:
I don't understand. It looks like it is only passed to `jedi#add_goto_window`?

Reply:
`show_goto_multi_results` is also called for `goto`, which has different modes.
It uses `mode` the for the qf title (prefix).